sonarqube/ocp-config/sonarqube.yml

```diff
@@ -441,3 +440,5 @@ parameters:
   description: "Sonarqube distribution url. Example community edition: https://sonarsource.bintray.com/Distribution/sonarqube/sonarqube-7.3.zip"
 - name: SONAR_VERSION
   description: "Sonarqube version specified in the variable SONAR_DISTRIBUTION_URL"
+- name: SONAR_AUTH_CROWD
+  description: "Set to true, if you want to use crowd as identity provider"
```

Comment:
Should we make this required? And looking at the other params, some things like `SONAR_VERSION` are also mandatory I guess?

Reply:
I agree. I would like to do that in a sepaate step, because, we should take a look at the parameters in general for the one config solution. Ok, with this?